service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -377,4 +405,109 @@ public static Direction createUnassignedDirection() {
                 .value(createCustomDirection(OTHERS))
                 .build());
     }
+
+    public static List<Element<Recital>> createRecitals() {
+        return ImmutableList.of(
+            Element.<Recital>builder()
+                .value(Recital.builder()
+                    .title("A title")
+                    .description("A description")
+                    .build())
+                .build()
+        );
+    }
+
+    public static Schedule createSchedule(boolean includeSchedule) {
```

Comment:
Maybe we could add this to separate helper (if we add all possible data creation here, it's going to grow in size a lot), and the whole helper thing should be moved to tests? Not in your pr but the place seems wrong.

Reply:
I think there is a PR to already do this